packages/notebook/src/actions.tsx

```diff
@@ -772,7 +772,7 @@ export namespace NotebookActions {
     const state = Private.getState(notebook);
 
     notebook.mode = 'command';
-    //Check if toBottom is true, if yes selection is made till bottom.
+    // Check if toBottom is true, if yes selection is made till bottom.
```

Comment:
Can you document the new parameter above in the docstring, below the line `@param notebook ...`

Reply:
Sure.